chain/chunks/src/test/multi.rs

```diff
@@ -187,10 +191,14 @@ fn test_distribute_chunk_track_all_shards() {
     });
     // Two network rounds is enough because each node should have
     // forwarded the parts to those block producers that need them.
-    // TODO: after phase 2, we will need a longer delay because validators
-    // that don't track the shard will not get forwards. We may also need
-    // to add the periodic resending handler.
-    test.run_for(NETWORK_DELAY * 2);
+    // However, after SingleShardTracking protocol upgrade, we need a longer
+    // delay because validators that don't track the shard will not get
+    // parts forwarded to them.
+    if cfg!(feature = "nightly") {
```

Comment:
Eh, well, statelessnet_protocol feature flag is not being used in a conformant way like nightly. There's a big script (fix_nightly_feature_flags.py) that ensures the nightly flag is being used properly, and statelessnet_protocol would fail that. You can take a look at the checks in there to see why it would. Anyway, the statelesssnet_protocol flag is fine for building neard, but if you're going to set up a CI for it, it might fail more than you expect.

Anyway, regarding the code suggestion that's actually very nice, I forgot I could just use PROTOCOL_VERSION :)